admin_manual/office/troubleshooting.rst

```diff
@@ -0,0 +1,41 @@
+===============
+Troubleshooting
+===============
+
+In case of connectivity issues, ensure that the following required connections are possible and not blocked by any firewall:
+
+- The users browser can reach both the Nextcloud Server as well as the Nextcloud Office server through HTTP(S)
+- The Nextcloud and the Nextcloud Office Server are using the same protocol
+- The Nextcloud server can reach the Nextcloud Office Server through HTTP(S)
+- The Nextcloud Office Server can reach the Nextcloud server through HTTP(S)
+
+Both the Nextcloud log as well as the Nextcloud Office Server log may reveal more detailed error messages in case of connection issues.
```

Comment:
I have to admit that I find it rather missleading to refer to the Collabora Online server as "Nextcloud Office Server". But here I really stumbled upon it. The log in question is the log by Collabora and I doubt that people who come to this page when trying to debug their setup will read the installation page (which says "There are several ways to run the coolwsd service, which is further referred to as Nextcloud Office Server.") first.

Given that the docs still talk about "Collabora Online" in many places (which is much clearer to me), I would suggest to use "Collabora Online Server" here as well.

Reply:
Yes, I think it makes sense to rather refer to it as Collabora Online server.

Reply:
I also adjusted that wording in the installation manual, the other one indeed causes just more confusion as there are more than enough different terms already anyway :D